unlock-app/src/components/interface/AppHeader.tsx

```diff
@@ -98,6 +102,32 @@ export const AppHeader = () => {
   const router = useRouter()
   const loginUrl = `/login?redirect=${encodeURIComponent(window.location.href)}`
 
+  const Links = ({ mobile = false }: LinksProps) => {
+    return (
+      <div
+        className={`${
+          mobile
+            ? 'absolute left-0 right-0 block h-auto px-10 pt-20 top-20'
+            : ''
+        } bg-ui-secondary-200`}
```

Comment:
I would avoid the prop here and directly use it inside container. I think it's fine if it's little verbose in class name.